### PR TITLE
changed from key error to value error in from_sbdb

### PR DESCRIPTION
**Pull request by Himanshu-Garg**
as per issue #609 
now when no. of objects are more than 1, instead of key error, its showing Value error as discussed in comments section of #609 

here is the new output ...
Screenshot (156)

pls review!!!

### CODE REVIEW

File: src/poliastro/tests/tests_twobody/test_orbit.py [hunk 1]
```diff
@@ -730,3 +730,10 @@ def test_from_sbdb():
         ss_classical = ss_target.classical()
 
         assert ss_classical == SBDB_DATA[target_name]
+
+
+def test_from_sbdb_raise_valueerror():
+    with pytest.raises(ValueError) as e:
+        Orbit.from_sbdb(name="Halley")
+
+    assert str(e.value)[0] == "2"
```
Comment (Juanlu001) [inline src/poliastro/tests/tests_twobody/test_orbit.py]: Can we test the full message?

File: src/poliastro/twobody/orbit.py [hunk 2]
```diff
@@ -469,31 +469,46 @@ def from_sbdb(cls, name, **kargs):
...
-            a,
-            ecc,
-            inc,
-            raan,
-            argp,
-            nu,
-            epoch=epoch.tdb,
-            plane=Planes.EARTH_ECLIPTIC,
-        )
+        except KeyError:
```
Comment (Juanlu001) [inline src/poliastro/twobody/orbit.py]: I don't like this way of using an error to control the flow. Perhaps could do instead:

```
if "count" in obj:
    # Handle the error
    raise ValueError(...)

# No need of else block, just do the rest of the stuff
a = obj["orbit"][...]
...
```

**Comment by Juanlu001** [comment]
@Himanshu-Garg are you still working on this?

File: src/poliastro/twobody/orbit.py (hunk 2 above)
Comment (Ivan9528) [reply src/poliastro/twobody/orbit.py]: I will take care of these changes asap! The idea is pretty much the same as in issue #609, right? Raise a proper message (not a key) when an error is produced.

Comment (Juanlu001) [reply src/poliastro/twobody/orbit.py]: Correct!

**Pull request closed without merging**